nemo_curator/modules/config.py

```diff
@@ -67,47 +67,67 @@ class FuzzyDuplicatesConfig(BaseConfig):
 
     # Minhash + LSH Config
     seed: int = 42
-    char_ngrams: int = 5
+    char_ngrams: int = 24
     num_buckets: int = 20
     hashes_per_bucket: int = 13
     use_64_bit_hash: bool = False
     buckets_per_shuffle: int = 1
 
-    false_positive_check: bool = True
+    false_positive_check: bool = False
     # Only required for fp check
```

Comment:
```suggestion
    # Only required for false positive check
```
Nit, but I prefer spelling out `false_positive` each time. Going to be making a similar comment about "fpp" on https://github.com/NVIDIA/NeMo-Curator/pull/285 but I think it can get pretty difficult to modify our codebase otherwise.